docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -1569,3 +1569,134 @@ Format mappers *must* have both `@PersistenceUnitExtension` and either `@JsonFor
 
 Having multiple JSON (or XML) format mappers registered for the same persistence unit will result in an exception, because of the ambiguity.
 ====
+
+[[jakarta-data]]
+== Static metamodel and Jakarta Data
+
+Both static metamodel and Jakarta Data capabilities of Hibernate ORM are available to the users
+through the `hibernate-jpamodelgen` annotation processor. Since it is an annotation processor,
+users are required to make an extra step of configuring the annotation processor in their build tool of choice:
```

Comment:
Don't ask me why, or where the rule is written, but in Quarkus documentation we're supposed to address directly to the reader.

```suggestion
you must configure the annotation processor in your build tool:
```